Automattic/jetpack
# Jetpack Visual Refresh: header and navigation

Fixes #29838

## Proposed changes:
* Update Jetpack header and navigation design

### Other information:


## Jetpack product discussion
p1HpG7-lx8-p2

## Does this pull request change what data or activity we track or use?
No.

## Testing instructions:
Visit the pages and confirm the header and top navigation look in accordance with the design (see #29838).

Some of the most important pages in both Desktop and Mobile modes:

- [ ] Jetpack Dashboard
- [ ] Modules
- [ ] License Activation
- [ ] Jetpack Dashboard under a non-admin user
- [ ] Jetpack Dashboard for non-connected secondary admin user
